scripts/gulp/lint.js

```diff
@@ -33,17 +34,28 @@ gulp.task( 'lint:build', function() {
  * Lints the test js files for errors.
  */
 gulp.task( 'lint:tests', function() {
-  return _genericLint( [
+  return _genericJSLint( [
```

Comment:
Trivial: We could follow generic-to-specific order here and have it be `_genericLintJs`.